doc/null_safety/null_safe_migration.md

```diff
@@ -51,9 +51,12 @@ dart pub global run over_react_codemod:null_safety_migrator_companion --yes-to-a
 ```
 
 This codemod will:
-- Add nullability hints to props/state that are defaulted/initialized in class components.
+- Add nullability hints to state mixin/class fields
   - These hints will cause defaulted/initialized values to be migrated as "late required". 
```

Comment:
Related to the above comment, I think it'd be worth calling out here that the same thing is done for props, but in the required props codemod